### PR TITLE
More EV charging data

### PR DESCRIPTION
**Pull request by erikbosch**
Originating from COVESA EV Charging project

### CODE REVIEW

**Comment by erikbosch** [comment]
Meeting notes: Please review

File: spec/Powertrain/TractionBattery.vspec [hunk 1]
```diff
@@ -378,6 +378,46 @@ Charging.ChargeVoltage.Phase3:
...
+  unit: kW
+  description: Maximum charging power of last or current charging event.
+
+Charging.EvseId:
+  datatype: string
+  type: sensor
+  description: EVSE charging point ID (without separators) of last or current charging event according to ISO 15118-2 Annex H.
+  comment: Length of id between 7 and 37 characters. ZZ00000 to be used if SECC cannot provide id
+
+Charging.Location:
```
[the remark this answers is not in the document]
Comment (erikbosch) [reply spec/Powertrain/TractionBattery.vspec:437]: Added it

**Review by adobekan — APPROVED** [review]
kk

**Comment by erikbosch** [comment]
Meeting notes: Ok to merge after @adobekan has reviewed last change

**Pull request merged**